scripts/explicit_translation.py

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
This is also a really useful script. Do you see this approach eventually replacing the current augur translate approach? We could update augur translate to alternately accept existing translations instead of inferred ancestral nucleotide sequences...

Reply:
`augur translate` has its value. the downside here is that if a sequence has a frameshifting deletion (sequence errors) it will be omitted from the nextalign translation. TreeTime will then infer it as the ancestral sequence. This is likely the best guess in most cases. but might not be the desired behavior in all cases. My suggestion would  be to use this script for a while and see when and where it is better/worse than `augur translate`. (we might also want to rename it :) )